mapper/mapper.go

```diff
@@ -419,7 +419,7 @@ func CreateReleaseCalendar(basePage coreModel.Page, params queryparams.Validated
 	calendar.Pagination.LimitOptions = []int{10, 25}
 	calendar.TotalSearchPosition = getTotalSearchPosition(currentPage, itemsPerPage)
 	calendar.Entries.Count = response.Breakdown.Total
-
+	calendar.RSSLink = "releasecalendar?rss&" + params.AsFrontendQuery().Encode()
```

Comment:
I prefer go's string formatter for concatenating strings

```suggestion
	calendar.RSSLink = fmt.Sprintf("releasecalendar?rss&%s", params.AsFrontendQuery().Encode())
```